onnxscript/ir/_schemas.py

```diff
@@ -0,0 +1,548 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Should we put a comment saying this should align to torch _schema.py until we move it to here?

Reply:
Actually, we only have the test for this file, which means that torch's _schema.py should be following this one.